src/docfx/build/page/Contributor.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
```

Comment:
put them into one model folder?

Reply:
lets do that during the data modeling casing change.